### PR TITLE
Explain how to make a continuum normalized spectrum

### PR DESCRIPTION
**Pull request by hamogu**
In the fitting section, the docs talk quite a bit about how continuum
normalized spectra are useful and how the tools help to make them.
However, it is not shown how to actually normalize the spectrum.
I opened #415 because I could not find the answer after experimenting.
This doc change adds a simple example.

### CODE REVIEW

File: docs/fitting.rst [hunk 1]
```diff
@@ -666,6 +666,10 @@ convenience functions to perform exactly this task.  An example is shown below.
     plt.title('Continuum Fitting')
     plt.grid('on')
 
+    # construct normalized spectrum by deviding the old spectrum
```
Comment (brechmos-stsci) [inline docs/fitting.rst]: deviding -> dividing

Comment (hamogu) [reply docs/fitting.rst]: @brechmos-stsci : Thanks! Done and force-pushed (this does not warrant a new commit).

File: docs/fitting.rst [hunk 2]
```diff
@@ -666,6 +666,10 @@ convenience functions to perform exactly this task.  An example is shown below.
     plt.title('Continuum Fitting')
     plt.grid('on')
 
+    # construct normalized spectrum by dividing the old spectrum
```
Comment (nmearl) [inline docs/fitting.rst]: This is a little buried with it being a comment in a code block. I think you should bring this out of the code block and demonstrate the results of the normalization with its own code block and plot.

Comment (hamogu) [reply docs/fitting.rst]: The problem is that you can't continue the code from one plot directive in another plot directive. If I regenerate a random spectrum a second time, it'll look different and we don't want the normalized spectrum to look different from the original one. Could set seed for random, but that's kind of not what we want. So, instead, I made it a second plot. the code is still in the same code block, but the fact that there is a second plot makes is a little more obvious to look there.

Comment (hamogu) [reply docs/fitting.rst]: Thank you for your advice. I thought I had a reasonable understanding of the plot directive given that I actually contributed a PR to matplotlib for one of the features, but apparently I was wrong. I will revise my doc change yet again.

Comment (hamogu) [reply docs/fitting.rst]: How about this version?
(Let me also note that there was nothing like this in the docs before. Even if my addition does not highlight it perfectly, I think it still makes the docs better.)

**Comment by hamogu** [comment]
I've restarted the Travis test, but get the same error on the Python 3.5 test, which I am *absolutely* certain has nothing to do with this PR (timeout at Zenodo).

Could someone please review with PR?

**Comment by hamogu** [comment]
@astropy/specutils-maintainers The failures in the Travis test is unrelated and intermittent (404 from Zenodo). Can someone please review this PR?

**Pull request merged**